third-party/rspamd/sample-configuration/extensions.properties

```diff
@@ -1 +1,2 @@
-guice.extension.module=org.apache.james.rspamd.module.RSpamDModule
\ No newline at end of file
+guice.extension.module=org.apache.james.rspamd.module.RSpamDModule
+guice.extension.task=org.apache.james.rspamd.module.RSpamDTaskExtensionModule
```

Comment:
Q: Do we still need `guice.extension.module=org.apache.james.rspamd.module.RSpamDModule` when we declared addtional route in `webadmin.properties` and DTO modules in `RSpamDTaskExtensionModule` ?

Reply:
yes.